items.json

```diff
@@ -3326,8 +3422,8 @@
     "weapons": []
   },
   {
-    "id": "crpg_arrows_v2_h0",
-    "baseId": "crpg_arrows_v2",
+    "id": "crpg_arrows_v3_h0",
+    "baseId": "crpg_arrows_v3",
     "name": "Simple Arrows",
```

Comment:
Items.Json still has the arrow changes

Reply:
reexported after reverting the arrow commit.